src/algorithms/math/fibonacci.js

```
@@ -0,0 +1,24 @@
+/**
+ * Show fibonacci sequence up to a given number
+ * @param {Number} num given number
+ */
+const fibonacci = (num) => {
+    let n1 = 0, n2 = 1;
+    let str = '';
+
+    if (num < 0) {
+        return 'Plz provide positive number';
```

Wouldn‘t it be more approriate to rather throw an error in this case?